dbms/src/Common/LooseBoundedMPMCQueue.h

```diff
@@ -21,9 +21,9 @@
 namespace DB
 {
 /** A simple thread-safe loose-bounded concurrent queue and basically compatible with MPMCQueue.
-  * Provide functions `forcePush` and `isFull` to support asynchronous writes.
+  * Provide functions `forcePush` and `isWritable` to support asynchronous writes.
```

Comment:
The comments confuse me a little bit, `asynchronous writes` is actually `non-blocking writes`?

Reply:
It is indeed a bit strange.. Changed to `Provide functions `forcePush` and `isWritable` to support non-blocking writes.`..